h/api/nipsa/models.py

```diff
@@ -0,0 +1,30 @@
+import sqlalchemy
+from sqlalchemy.orm import exc
+
+from h.api import db
+
+
+class NipsaUser(db.Base):
+
+    """A NIPSA entry for a user (SQLAlchemy ORM class)."""
+
+    __tablename__ = 'nipsa'
+    userid = sqlalchemy.Column(sqlalchemy.UnicodeText, primary_key=True,
+                               index=True)
```

Comment:
AIUI there's nothing gained by adding an index on a column that already has a PRIMARY KEY constraint.